padertorch/contrib/examples/pit/train.py

```diff
@@ -133,12 +118,14 @@ def init(_config, _run):
 
 
 @ex.capture
-def prepare_and_train(_config, _run, train_dataset, validate_dataset):
+def prepare_and_train(_config, _run, train_dataset, validate_dataset, db_path):
```

Comment:
Same comment as in evaluate: We might want to ave the same name for `db_path` in here and the DPRNN example